packages/devtools_app/lib/main.dart

```diff
@@ -4,21 +4,31 @@
 
 import 'package:flutter/material.dart';
 import 'package:flutter/foundation.dart';
-import 'package:html_shim/html.dart' show document;
+import 'package:html_shim/html.dart' show document, window;
 
 import 'src/flutter/app.dart';
+import 'src/framework/framework_core.dart';
 
 void main() {
+  String url;
   if (kIsWeb) {
     // Clear out the unneeded HTML from index.html.
     for (var element in document.body.querySelectorAll('.legacy-dart')) {
       element.remove();
     }
+    url = window.location.toString();
   } else {
     // When running in a desktop embedder, Flutter throws an error because the
     // platform is not officially supported. This is not needed for web.
     debugDefaultTargetPlatformOverride = TargetPlatform.fuchsia;
+    // TODO(jacobr): we don't yet have a direct analog to the URL on flutter
+    // desktop.
+    // Hard code to the dark theme as the majority of users are on the dark
+    // theme.
+    url = 'http://127.0.0.1/?theme=dark';
```

Comment:
Suggestion: make this '/'.  This side of the app isn't on the web, and pointing to localhost doesn't actually mean anything.  This will also match with the app's initial route (more or less).

Reply:
done